### ISSUE TITLE
nginx ssl config 

### ISSUE THREAD

**Issue by nsdonner** [eveseat/docker-eveseat-nginx#1]
It seems to that ssl doesn't configured correct. When i tried to access seat via https there are errors in docker logs about missed certificate. It would not be a big problem with revers proxy in frontend, but app generates internal links with http instead https, so it useless.

**Comment by MrBMT** [comment]
If you're using SSL, you need to set it in your .env file too, else it won't have any effect.

**Comment by nsdonner** [comment]
may i see an example? I'm not very in with docker, and i didn't find it in seat docs.

**Comment by MrBMT** [comment]
Sorry, I missed that you were using docker.

I believe you'll need to make the configuration changes as instructed [here](https://eveseat.github.io/docs/guides/installation/manual_installation/docker/#configuration-changes), unfortunately I can't give a full example as I don't use the docker install myself, but the APP_URL and EVE_CALLBACK_URL are the ones you need to change to ensure you're using HTTPS.

**Comment by nsdonner** [comment]
yep, i've already done with this laravel configs, but it don't solves nginx cert misconfig

**Comment by leonjza** [comment]
I forgot to disable issues for this repo. @nsdonner mind relogging this one in the main repo? https://github.com/eveseat/seat
